docs/notebooks/customize.ipynb

```diff
@@ -149,14 +149,23 @@
    "source": [
     "This method checks that there are less than `nm=11` invalid values in a month and that there are no consecutive runs of `nc>=5` invalid values. Thus, every month is now valid.\n",
     "\n",
-    "Finally, it is possible for advanced users to register their own method. Xclim's missing methods are in fact based on class instances. Thus, to create a custom missing class, one should implement a subclass based on `xclim.core.checks.MissingBase` and overriding at least the `is_missing` method. The method should take a `null` argument and  a `count` argument.\n",
+    "<div class=\"alert alert-warning\">\n",
```

Comment:
I was under the impression that Jupyter supported Markdown ways of making warning boxes, e.g.
```
```{warning}
```

Reply:
Oh! I simply copied code over from another notebook...

Reply:
Hah, well that's fine too. Something else we should verify, I imagine.